chia/rpc/rpc_server.py

```diff
@@ -101,14 +101,14 @@ async def get_connections(self, request: Dict) -> Dict:
         request_node_type: Optional[NodeType] = None
         if "node_type" in request:
             request_node_type = NodeType(request["node_type"])
-        if self.rpc_api.service.server is None:
+        if self.rpc_api.data_layer_service.server is None:
```

Comment:
@almogdepaz, it looks like maybe a refactor rename reached a bit far out?  Though I think maybe this code expects the RPC implementations to use `.service`?

Reply:
I suppose I ought to link to an example error in the CI runs.

https://github.com/Chia-Network/chia-blockchain/pull/8492/checks?check_run_id=3646582478#step:14:729
```
rpc_api = <chia.rpc.full_node_rpc_api.FullNodeRpcApi object at 0x7f0400847370>
self_hostname = 'localhost', daemon_port = 55400, rpc_port = 51000
stop_cb = <bound method Service.stop of <chia.server.start_service.Service object at 0x7f0401b15220>>
root_path = PosixPath('/tmp/tmpt5azhi35')
net_config = {'ALERTS_URL': 'download.chia.net/notify/mainnet_alert.txt', 'CHIA_ALERTS_PUBKEY': '89b7fd87cb56e926ecefb879a2...a': {'crt': 'config/ssl/ca/chia_ca.crt', 'key': 'config/ssl/ca/chia_ca.key'}, 'daemon_max_message_size': 50000000, ...}
connect_to_daemon = False

    async def start_rpc_server(
        rpc_api: Any,
        self_hostname: str,
        daemon_port: uint16,
        rpc_port: uint16,
        stop_cb: Callable,
        root_path: Path,
        net_config,
        connect_to_daemon=True,
    ):
        """
        Starts an HTTP server with the following RPC methods, to be used by local clients to
        query the node.
        """
        app = aiohttp.web.Application()
        rpc_server = RpcServer(rpc_api, rpc_api.service_name, stop_cb, root_path, net_config)
>       rpc_server.rpc_api.data_layer_service._set_state_changed_callback(rpc_server.state_changed)
E       AttributeError: 'FullNodeRpcApi' object has no attribute 'data_layer_service'
```